corehq/apps/cloudcare/views.py

```diff
@@ -471,20 +472,31 @@ def put(self, request, *args, **kwargs):
 def report_formplayer_error(request, domain):
     data = json.loads(request.body)
     error_type = data.get('type')
+    referer = request.META.get('HTTP_REFERER', '')
+
+    if '/cloudcare/apps/v2/' in referer:
+        origin = 'web_apps'
+    elif '/cloudcare/apps/preview_app/' in referer:
+        origin = 'app_preview'
+    else:
+        origin = 'unknown'
+
     if error_type == 'webformsession_request_failure':
         metrics_counter('commcare.formplayer.webformsession_request_failure', tags={
             'request': data.get('request'),
             'statusText': data.get('statusText'),
             'state': data.get('state'),
             'status': data.get('status'),
             'domain': domain,
+            'origin': origin,
         })
         notify_error(message='Formplayer: request failure in web form session', details=data)
     elif error_type == 'show_error_notification':
         message = data.get('message')
         metrics_counter('commcare.formplayer.show_error_notification', tags={
             'message': _message_to_tag_value(message or 'no_message'),
             'domain': domain,
+            'origin': origin,
         })
         notify_error(message=f'Formplayer: showed error to user: {message}', details=data)
```

Comment:
I'm also considering changing the prefix here. I noticed after looking at it in sentry that takes up way too much off the allotted space, so that we're not able to see most of the actual error message: e.g. https://sentry.io/organizations/dimagi/issues/1653009121/?environment=production&project=136860&query=is%3Aunresolved+Formplayer&statsPeriod=14d.

I'm accepting suggestions. I'm currently considering "[WA/AP] red banner:", and maybe replacing the "Formplayer:" in the other two sentry messages with "[WA/AP]". Not all of the errors are properly "formplayer" errors, and "Web Apps/App Preview" is too long. "[Cloudcare]" is also longer, but maybe that's better than [WA/AP]?

Reply:
`[WA/AP]` is rather opaque. If "formplayer" isn't accurate enough, "[Cloudcare]" seems right to me. I can't quite tell from the comment if you're also planning to delete the "showed error to user" and "request failure in web form session" text, but I think you should delete that altogether, or put it in the detailed data if you think it's going to be valuable.